lang/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  Search:
+    INPUT_LABEL: false
+    INPUT_ARIALABEL: 'Enter search terms'
+    INPUT_PLACEHOLDER: 'Enter search terms'
+    INPUT_TITLE: 'Search'
+    BUTTON_LABEL: '<i class="fal fa-search"></i>'
```

Comment:
Good point, what if we just had two config options:
- UseIcon <- boolean
- IconTag <- string

Then, if `UseIcon` is false, it uses whatever `BUTTON_LABEL` is appropriate for the user's language.

Reply:
I think I want to go the most flexible way, no configuration options, keep the language support:
- This is really just a design/style choice that the module does not need to facilitate directly, it should just be possible though to achieve 
- Add extends to the form and form process functions - Fields can be added/overridden/removed/updated
- The form template could be overridden if needed
- We update the default string to be "Search" - the button icon option would need to be defined on the project side

I think this is the ideal way to set this up, instead of adding a bunch of configuration options that just complicate things